gazebo_ros/scripts/spawn_entity.py

```diff
@@ -256,11 +256,20 @@ def entity_xml_cb(msg):
         if self.args.bond:
             self.get_logger().info(
                 'Waiting for shutdown to delete entity [{}]'.format(self.args.entity))
+
+            self.clientDeletion = self.create_client(
+                DeleteEntity, '%s/delete_entity' % self.args.gazebo_namespace)
+            if not self.clientDeletion.wait_for_service(timeout_sec=self.args.timeout):
+                self.get_logger().error(
+                    'Service %s/delete_entity unavailable. ' +
+                    'Was Gazebo started with GazeboRosFactory?')
```

Comment:
This format string is missing `self.args.gazebo_namespace`, as is the original code. This PR will likely conflict with #1479, which fixes two occurrences of that bug.